std_msgs/Header header
Status status

float32 north_position_rms
float32 east_position_rms
float32 down_position_rms
float32 north_velocity_rms
float32 east_velocity_rms
float32 down_velocity_rms
float32 roll_rms
float32 pitch_rms
float32 heading_rms
